### PR TITLE
erasure-code: enable isa-l for aarch64

### PR DESCRIPTION
**Pull request by egggHang**
Signed-off-by: Hang Li <lihang48@hisilicon.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by fengchunsong** [comment]
这种一般需要给出性能优化数据，用不用的性能差异

**Comment by egggHang** [comment]
@fengchunsong 这个patch只涉及arm版本的使能，不含性能优化相关

**Comment by wade-wu** [comment]
> @fengchunsong 这个patch只涉及arm版本的使能，不含性能优化相关

可以把数据加一下，未来便于推动社区采纳及更新ISAL版本

File: src/erasure-code/isa/CMakeLists.txt [hunk 1]
```diff
@@ -57,6 +58,34 @@ set(isa_srcs
   ErasureCodePluginIsa.cc
   xor_op.cc
 )
+elseif(HAVE_AARCH64_ASM_ELF64)
+set(isa_srcs
```
Comment (rosinL) [inline src/erasure-code/isa/CMakeLists.txt]: need indent here

File: src/erasure-code/isa/CMakeLists.txt [hunk 2]
```diff
@@ -2,61 +2,90 @@
...
+    ${isal_src_dir}/erasure_code/aarch64/ec_multibinary_arm.S
+    ErasureCodeIsa.cc
+    ErasureCodeIsaTableCache.cc
+    ErasureCodePluginIsa.cc
+    xor_op.cc
+  )
+  set_source_files_properties(
+    ${isal_src_dir}/erasure_code/aarch64/ec_multibinary_arm.S
+    PROPERTIES COMPILE_FLAGS "-D__ASSEMBLY__"
 )
```
Comment (rosinL) [inline src/erasure-code/isa/CMakeLists.txt]: indent

Comment (egggHang) [reply src/erasure-code/isa/CMakeLists.txt]: modified

**Pull request closed without merging**